release/rtool.py

```diff
@@ -96,7 +76,7 @@ def homebrew_pr():
     subprocess.check_call([
         "brew",
         "bump-formula-pr",
-        "--url", "https://github.com/mitmproxy/mitmproxy/archive/v{}".format(get_version()),
+        "--url", "https://github.com/mitmproxy/mitmproxy/archive/v{}.tar.gz".format(get_version()),
         "mitmproxy",
```

Comment:
In the past we invoked rtool via tox - but this seems to be missing now (was replaced by the ci script via tox).
I think the whole rtool can go now, or whats the plan?